install/aws/aws-cloudwatch-metric-streams/install.yml

```diff
@@ -0,0 +1,13 @@
+id: aws-cloudwatch-metric-streams
+name: AWS-CloudWatch-Metric-Streams
+title: AWS Metric Streams
+description: |
+  AWS CloudWatch Metric Streams integration enables streaming of metrics from any AWS service (including custom namespaces) to New Relic. 
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream/
```

Comment:
please Use the proper YAML formatting >- for documentation URL references.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation